### ISSUE TITLE
Task: Identification of Wrong Examples

### ISSUE THREAD

**Issue by caranha** [amanotaiga/Tensorflow_CNN#7]
Change your learning/testing scripts to identify which examples your neural network gets wrong. The script should (optionally, by parameter) create a "errors" directory where it store samples that were misclassified. Do not forget to set the true label and predicted label for these samples somewhere (i.e. write in the image itself or in the filename)
